flytekit/remote/executions.py

```diff
@@ -105,11 +106,16 @@ def __init__(
         **kwargs,
     ):
         super(FlyteWorkflowExecution, self).__init__(*args, **kwargs)
-        self._node_executions = None
+        self._node_executions: Optional[Dict[str, FlyteNodeExecution]] = None
         self._flyte_workflow: Optional[FlyteWorkflow] = None
         self._remote = remote
         self._type_hints = type_hints
 
+    def traverse_node_executions(self, exclude_start_end_nodes=True):
```

Comment:
Can we add a type hint to this?  `Generator[FlyteNodeExecution, None, None]` I think.